pennylane/logging/configure.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
```

Comment:
I'm not too keen on this filename. Also, can we not place these into the init file for now, since it is a minor set of content?

Reply:
OK. Choose a name and we can go with it. What do you mean by not putting it? Could you please be more specific?

Reply:
Sorry, colloquialisms. Actually, let's name this `config.py` and we can update later after the merge pending discussions with Core.

Reply:
Sure thing!